src/components/image-config/index.test.tsx

```diff
@@ -0,0 +1,96 @@
+import { render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+
+import ImageConfig from '.'
+
+const testImage = (
```

Comment:
yay, ty for writing these tests 🥳

Reply:
No problem, if by "writing" you mean "copypasta-ing from `learn`" 😝